composer.json

```diff
@@ -14,36 +15,37 @@
         "ext-pdo": "*",
         "ext-zip": "*",
         "doctrine/common": "^2.0.0",
-        "ergonode/account": "^1.0.0",
-        "ergonode/api": "^1.0.0",
-        "ergonode/attribute": "^1.0.0",
-        "ergonode/authentication": "^1.0.0",
-        "ergonode/category": "^1.0.0",
-        "ergonode/channel": "^1.0.0",
-        "ergonode/comment": "^1.0.0",
-        "ergonode/completeness": "^1.0.0",
-        "ergonode/condition": "^1.0.0",
-        "ergonode/core": "^1.0.0",
-        "ergonode/designer": "^1.0.0",
-        "ergonode/editor": "^1.0.0",
-        "ergonode/es": "^1.0.0",
-        "ergonode/exporter": "^1.0.0",
-        "ergonode/exporter-file": "^1.0.0",
-        "ergonode/fixture": "^1.0.0",
-        "ergonode/grid": "^1.0.0",
-        "ergonode/importer": "^1.0.0",
-        "ergonode/mailer": "^1.0.0",
-        "ergonode/migration": "^1.0.0",
-        "ergonode/multimedia": "^1.0.0",
-        "ergonode/notification": "^1.0.0",
-        "ergonode/product": "^1.0.0",
-        "ergonode/product-collection": "^1.0.0",
-        "ergonode/reader": "^1.0.0",
-        "ergonode/segment": "^1.0.0",
-        "ergonode/shared-kernel": "^1.0.0",
-        "ergonode/transformer": "^1.0.0",
-        "ergonode/value": "^1.0.0",
-        "ergonode/workflow": "^1.0.0",
+        "ergonode/account": "^1.0.0@beta",
```

Comment:
It's not forcing the update of all packages. Not sure if it's good practice tbh. Since beta's are not fully compatible may cause issues